src/pages/Admin/Admin_v2.tsx

```diff
@@ -3,30 +3,30 @@ import "firebase/storage";
 import dayjs from "dayjs";
 import advancedFormat from "dayjs/plugin/advancedFormat";
 
-// Components
-import AuthenticationModal from "components/organisms/AuthenticationModal";
-import VenueDetails from "./Venue/Details";
+import { Venue_v2 } from "types/venues";
+
+import { orderedVenuesSelector } from "utils/selectors";
 
-// Hooks
 import { useSelector } from "hooks/useSelector";
 import { useUser } from "hooks/useUser";
 import useRoles from "hooks/useRoles";
 import { useIsAdminUser } from "hooks/roles";
+import { useAdminVenues } from "hooks/useAdminVenues";
+import { useVenueId } from "hooks/useVenueId";
 
-// Styles
-import "./Admin.scss";
-import { Venue_v2 } from "types/venues";
 import { AuthOptions } from "components/organisms/AuthenticationModal/AuthenticationModal";
+import { AdminVenues } from "components/organisms/AdminVenues/AdminVenues";
+import AuthenticationModal from "components/organisms/AuthenticationModal";
+import BasicInfo from "./BasicInfo";
 import AdminSidebar from "./Sidebar/Sidebar";
-import { useVenueId } from "hooks/useVenueId";
-import { orderedVenuesSelector } from "utils/selectors";
-import { useAdminVenues } from "hooks/useAdminVenues";
 import EntranceExperience from "./EntranceExperience";
 import AdvancedSettings from "./AdvancedSettings";
+import VenueDetails from "./Venue/Details";
 // import TicketingAndAccess from "./TicketingAndAccess";
```

Comment:
Is this needed here anymore? Can it be removed?

Reply:
Some venues have tickets and this will be added in the future when we have the whole picture.

Reply:
Makes sense. It might be useful to add a `// @debt` comment above it explaining that. Otherwise I'll end up asking about it in every PR I see.

Reply:
Makes sense. It might be useful to add a `// @debt` comment above it explaining that. Otherwise I'll end up asking about it in every PR I see.